packages/elements/src/datetime-field/utils.ts

```diff
@@ -0,0 +1,87 @@
+import { DateTimeFormatPart, DateTimeFormatPartTypes, InputSelection } from './types';
+import { Direction } from './constants.js';
+
+const IterablePartTypes: DateTimeFormatPartTypes[] = ['year', 'day', 'month', 'hour', 'minute', 'second', 'weekday', 'dayPeriod', 'fractionalSecond'];
+export const isAllowedPart = (part: DateTimeFormatPart): boolean => IterablePartTypes.includes(part.type);
```

Comment:
At the moment, this is only used internally on `packages/elements/src/datetime-field/utils.ts` so this might not even need an export.

Reply:
sorry, I did not get you?

Reply:
I meant that we don't have to export `isAllowedPart` if we do not used it anywhere else.

Reply:
got you, done